starfish/stack.py

```diff
@@ -608,7 +609,8 @@ def _iter_tiles(
             yield array
 
     def apply(
-            self, func, is_volume=False, in_place=True, verbose: bool=False, **kwargs
+            self, func, is_volume=False, in_place=True, verbose: bool=False, multiprocess=True,
```

Comment:
so rather than say multiprocess=True (or False), perhaps you should do: max_workers=None, where None=fork off as many as you have cores.